device-types/Ciena/8190.yaml

```diff
@@ -0,0 +1,138 @@
+---
+manufacturer: Ciena
+model: '8190'
+slug: ciena-8190
+part_number: 170-8190-900
+u_height: 2
+is_full_depth: false
+weight: 22
+weight_unit: kg
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it